client/src/App.tsx

```diff
@@ -9,6 +9,16 @@ const App = () => {
     const [messageCount, setMessageCount] = useState<number>(5);
     const [messagePosition, setMessagePosition] = useState<number>(1);
     const [messageDisappearTime, setMessageDisappearTime] = useState<number>(5000);
+    const [eventSource, setEventSource] = useState<EventSource | null>(null);
+
+    useEffect(() => {
+        const source = new EventSource('http://localhost:9000/events');
+        setEventSource(source);
+
+        return () => {
+            source.close();
+        };
+    }, []);
```

Comment:
Initialized the EventSource connection in the root component. The EventSource connection is closed when the App component unmounts.